packages/forms/src/types.js

```diff
@@ -26,8 +27,12 @@ type TableSchema = {
 
 type Schema = Array<TableSchema>;
 
-type SchemaContextValue = {
-  schema: Schema,
+type SchemaContextValue = Schema;
+
+type RenderableProps = {
+  component?: React.ComponentType<*>,
```

Comment:
`React.ComponentType<*>` means that all components in this prop should be the same interfaces. Is this expected behavior?

Reply:
@ej9x fixed